packages/cysync-core/src/pages/MainApp/Components/AssetAllocation.tsx

```diff
@@ -0,0 +1,171 @@
+import {
+  AllocationShare,
+  AssetIconNameBox,
+  NameBox,
+  NameVariants,
+  TableBody,
+  TableBox,
+  TableBoxDataRow,
+  TableBoxHeader,
+  TableBoxHeaderData,
+  TableBoxTitle,
+  Typography,
+} from '@cypherock/cysync-ui';
+import React, { useEffect, useState } from 'react';
+
+interface HeadersData {
+  name: string;
+  width?: string;
+  padding?: string;
+  $noFlex?: boolean;
+}
+
+const headersData: HeadersData[] = [
+  {
+    name: 'Asset',
+    padding: '16px 20px 16px 96px',
+    width: '300',
+    $noFlex: true,
+  },
+  {
+    name: 'Price',
+  },
+  {
+    name: 'Amount',
+  },
+  {
+    name: 'Value',
+  },
+  {
+    name: 'Allocation',
+    width: '300',
+    $noFlex: true,
+  },
+];
+
+interface AssetDataType {
+  name: NameVariants;
+  symbol: string;
+  price: string;
+  amount: string;
+  value: string;
+  allocation: number;
+}
+
+export const AssetAllocation = () => {
+  const [sortedBy, setSortedBy] = React.useState(headersData[0].name);
+  const [ascending, setAscending] = useState(true);
+  const [data, setData] = useState<AssetDataType[]>([
+    {
+      name: 'Bitcoin',
+      symbol: 'BTC',
+      price: '$ 16981.44',
+      amount: '0.0178 BTC',
+      value: '$2.9827',
```

Comment:
Amounts should not contain units, because that way: 
- We can write a generic sorting algorithm.
- Add unit only when displaying
```suggestion
      price: '16981.44',
      amount: '0.0178',
      value: '2.9827',
```